docs/rules/void-dom-elements-no-children.md

```diff
@@ -1,4 +1,4 @@
-# Prevent void DOM elements (e.g. `<img />`, `<br />`) from receiving children
+# Prevent void DOM elements (e.g. `<img />`, `<br />`) from receiving children (react/void-dom-elements-no-children)
```

Comment:
This was my request, per https://github.com/yannickcr/eslint-plugin-react/pull/1078#discussion_r101881565